amworkflow/occ_helpers.py

```diff
@@ -191,3 +206,245 @@ def topo_explorer(shape: TopoDS_Shape, shape_type: str) -> list:
         result.append(explorer.Current())
         explorer.Next()
     return result
+
+
+def intersect(item: TopoDS_Shape, position: float, axis: str) -> TopoDS_Shape:
+    """
+    @brief Returns the topo shape intersecting the item at the given position.
```

Comment:
can we agree on using the style Params: and Returns: for the doc strings

Reply:
Yes we can. 

I am wondering if we can adopt sphinx style docstring:

```python
"""[Summary]

:param [ParamName]: [ParamDescription], defaults to [DefaultParamVal]
:type [ParamName]: [ParamType](, optional)
...
:raises [ErrorType]: [ErrorDescription]
...
:return: [ReturnDescription]
:rtype: [ReturnType]
"""
```

I think lots of new codes I wrote are documented using this style. But if you don't like it I can change them to apply to Params: and Returns:

I suggests to do this in a new issue since there are tons of docstrings now waiting to convert. What do you think